.env.stg

```diff
@@ -10,4 +10,6 @@ GATSBY_FIREBASE_STORAGE_BUCKET="tally-stg.appspot.com"
 GATSBY_FIREBASE_MESSAGING_SENDER_ID="110043097074"
 GATSBY_FIREBASE_APP_I="1:110043097074:web:48cfac48940f26cc912d6b"
 
-GATSBY_DISCORD_APP_ID="1008427467918295044"
\ No newline at end of file
+GATSBY_DISCORD_APP_ID="1008427467918295044"
+
+POSTHOG_API_KEY="phc_VzveyNxrn2xyiKDYn7XjrgaqELGeUilDZGiBVh6jNmh"
```

Comment:
I think you will need a `GATSBY_` prefix, for the variable to be available in prod build